station_data.py

```diff
@@ -4,8 +4,21 @@
 from StringIO import StringIO
 from numpy import genfromtxt
 import ast
+import re
```

Comment:
It might! This was the first method I tried. I don't know how you parse JSON. We will accept the pull request as is to enable getting the station ids, then you can improve as you like.